sdk/identity/identity-broker/README.md

```diff
@@ -19,6 +19,14 @@ useIdentityPlugin(nativeBrokerPlugin);
 
 - An [Azure subscription](https://azure.microsoft.com/free/nodejs/).
 
+> Note: For local development with `@azure/identity-broker`, you may need to install additional tools. [node-gyp](https://github.com/nodejs/node-gyp) is used to compile [addons](https://nodejs.org/api/addons.html) for accessing system APIs. Installation requirements are listed in the [node-gyp README](https://github.com/nodejs/node-gyp#installation).
+
+On Linux, the dependency library uses `libsecret` so you may need to install it. Depending on your distribution, you will need to run the following command:
```

Comment:
I'm not sure what's meant by "dependency library" in this context.

Reply:
It's referring to `@azure/msal-node-extensions` because that's being used by a dependency there. I think we can change it to `the library uses` so we don't have to surface the dependency structure to `keytar`. Would that work?